data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -490,4 +490,62 @@
         icon_completed="data/campaigns/Heir_To_The_Throne/images/units/konrad-lord.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
         limit=24
     [/achievement]
+
+    # Kalenz was born and raised in Lintair
+    # so chose Lintair over Wesmere
+    # otherwise, this was supposed to be titled "Legend of Wesmere"
+    [achievement]
+        id="HttT_Hero_of_Lintair"
+        name= _ "Hero of Lintair"
```

Comment:
Somehow I thought it was "Lintanir"…? Was I wrong?

Reply:
It is Lintair Forest or in elf-tongue, "Lins-Elen"..
Should the achievement title be in North elven tongue as well?

That is, *Callo o- Lins-Elen*?

Reply:
"Lintair" appears in exactly one place in mainline.

There are 32 occurrences of "Lintanir".

Seems pretty clear to me that it should be "Lintanir" and the one occurrence of "Lintair" is a typo.

Reply:
```ini
[message]
            speaker=Kalenz
            message= _ "Very well. If you convince him not to raise his sword against us, the Lintair elves shall grant him safe passage."
        [/message]
```

Stop. I trust Kalenz, It's Lintair forest and everywhere else has the typo.

Reply:
This was most likely a typo that has been removed since, so it should be Lintanir.